Source/Core/Core/IOS/USB/Bluetooth/RealtekFirmwareLoader.cpp

```diff
@@ -158,7 +169,6 @@ bool IsKnownRealtekBluetoothDevice(u16 vid, u16 pid)
       {0x04c5, 0x161f},
       {0x0b05, 0x18ef},
       {0x13d3, 0x3548},
-      {0x13d3, 0x3549},
```

Comment:
Note, this exact ID is already present in the "Realtek 8822CU Bluetooth devices" section at the top, this one I removed  here was a duplicate